files/common/lib/open-iscsi/zpool_on_iscsi.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 by Delphix. All rights reserved.
+#
+
+#
+# This script checks if any iSCSI devices are in use by an active zfs pool.
```

Comment:
looks like shfmt doesn't care, but personally, I'd prefer to add the "closing" `#` to this comment, to match the "opening" `#`.